src/index.html

```diff
@@ -1,17 +1,98 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8"> </meta>
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    ></meta>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge"></meta>
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.css"></link>
   </head>
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form method="post"></form>
+    
+    <div class="box1">
+      <div class="form-field">
+        <label for="Surname"> Surname: </label>
+        <input type="text" name="Surname"
+         id="Surname">
+      </div>
```

Comment:
there's no real need for a `<div>` wrapper for each input;
if you need a block behavior - define `display: block` in css and apply class directly to `<label>`;
you're missing `autocomplete="off"` for text inputs as per readme;
it's reasonable to contain input inside label;
multi-line attributes when there are more than 2;
```suggestion
        <label for="Surname" class="form-field">
          Surname:
          <input
            class="form-field"
            type="text"
            name="Surname"
            id="Surname"
            autocomplete="off"
          >
      </label>
```

Reply:
Hi, many thanks for the feedback. Please review and let me know how to solve the remaining variances to the reference file.  Also, whenever I use "git commit" command, it sometimes is not possible to leave a comment, is there a trick?

Many thanks!